### PR TITLE
subscribe component

### PR DESCRIPTION
**Pull request by DineshBabu1989**
Review the component

### CODE REVIEW

File: src/components/EmbeddedServices.css [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+$font1: 'Lato', sans-serif;
+*/
+.show-grid-fluid-embedded-services {
+  background-color: #f7f7f7; }
+
+.show-grid-embedded-services {
+  background-color: #f7f7f7; }
+
+.show-title-col {
+  background-color: #f7f7f7;
```
Comment (rajeshdavidbabu) [inline src/components/EmbeddedServices.css:23]: Move all the color codes to a single .sass file and refer it from there

File: src/components/EmbeddedServices.jsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Clearfix , Grid, Row, Col, Image, Button } from 'react-bootstrap';
+import "./EmbeddedServices.css";
+
+
+
+const EmbeddedServices =  () => {
+    
+return(
```
Comment (rajeshdavidbabu) [inline src/components/EmbeddedServices.jsx:9]: use proper indendations

File: src/sass/components/EmbeddedServices.scss [hunk 3]
```diff
@@ -0,0 +1,61 @@
+@import "../Variables";
+@import "../mixins";
+
+/*$themeColor : #112a39!important;
```
Comment (rajeshdavidbabu) [inline src/sass/components/EmbeddedServices.scss:4]: remove all the commented code change

**Review by rajeshdavidbabu — COMMENTED** [review]
Make your code look beautiful. Not ugly

File: src/components/EmbeddedServices.css (hunk 1 above)
Comment (DineshBabu1989) [reply src/components/EmbeddedServices.css:23]: i did it

File: src/components/EmbeddedServices.jsx (hunk 2 above)
Comment (DineshBabu1989) [reply src/components/EmbeddedServices.jsx:9]: all the new files are indented

**Pull request closed without merging**